test/unit/range/views/drop_test.cpp

```diff
@@ -33,29 +32,27 @@
 template <typename adaptor_t>
 void do_test(adaptor_t const & adaptor, std::string const & vec)
 {
+    using namespace std::string_literals;
+
     // pipe notation
-    auto v = vec | adaptor(3);
-    EXPECT_EQ("bar", std::string(v));
+    EXPECT_RANGE_EQ("bar"s, vec | adaptor(3));
```

Comment:
why not `"bar"sv` ?

Reply:
I started to use it in the meantime. I can change this here too. I didn't want to go a second time through all.